source/common/protobuf/utility.cc

```diff
@@ -582,6 +582,32 @@ bool ValueUtil::equal(const ProtobufWkt::Value& v1, const ProtobufWkt::Value& v2
   }
 }
 
+ProtobufWkt::Value ValueUtil::stringValue(const std::string& str) {
```

Comment:
Do you mind dropping a todo into the access log formatter code to switch to these helpers (or if you want extra karma points, convert it to use these)?

Reply:
Sure!

Reply:
https://github.com/envoyproxy/envoy/pull/9432/commits/4598ff29573e714883a83e98801ff205bbe49655